clients/admin-ui/src/types/api/README.md

```diff
@@ -0,0 +1,27 @@
+# FidesCtl API Types
+
+**Do not manually edit these files.**
+
+The TypeScript files in this directory are auto-generated by OpenAPI. This ensures our frontend uses the same schema as the backend without needing to manually convert Python classes to TypeScript. Currently, we only generate model types.
```

Comment:
I put these generated files under `~/types`  because as long as we're only generating the models, these will only be type defs. I would be surprised if we ever wanted to use the networking code this can generate, but if we did it wouldn't really belong under "types".